Update README (#29)
ONSdigital/dp-legacy-cache-api: README.md

# dp-legacy-cache-api

REST API for managing cache control information for pages within the legacy CMS. dp-legacy-cache-api is called by:
- dp-legacy-cache-proxy to read the release time for a particular item of content
- Zebedee to set the right cache time for content based on publish notifications

> | Database Fields              | 

Reviewer: It's not clear at this point that "the api talks to documentdb/mongodb(?), in the _<name_of_collect>_ collection" so might be worth mentioning that.

- these fields are in documentdb or mongodb collections (which collection?), running in the env.?
- what does `ID` in the next line refer to - ID of _what_, exactly (page? release?)?
- `path` refers to the path (URI) of the published page?

# dp-legacy-cache-api

REST API for managing cache control information for pages within the legacy CMS. dp-legacy-cache-api is called by:
- dp-legacy-cache-proxy to read the release time for a particular item of content (Web subnet/ read-only mode)
- Zebedee to set the right cache time for content based on publish notifications (Publishing subnet)

> The API talks to DocumentDB in sandbox/staging and MongoDB locally in the cachetimes collection.

Reviewer: ```suggestion
The API talks to DocumentDB in the environment (or MongoDB locally) in the `cachetimes` collection.
```

> | Database Fields                      | 

Reviewer: can these descriptions be put into another column, please

| IS_PUBLISHING                | false                           | Determines if the instance is in publishing or not                                                                 |
| ZEBEDEE_URL                  | http://localhost:8082           | Zebedee host address and port for authentication                                                                   |

### Secrets are Auto-Deployed

> When the secrets are deployed to Vault, this will automatically cause Nomad to trigger a redeployment of the application to pick up the new secrets.

Reviewer 2: Is this correct? This is new behaviour that I'm unaware of so just checking.

> ### Secrets are Auto-Deployed
> When the secrets are deployed to Vault, this will automatically cause Nomad to trigger a redeployment of the application to pick up the new secrets. Please note that this functionality does not appear to work with the current nomad/vault versions, but if this is upgraded it may then be functional. 
> 

Reviewer 2: The current headline and wording still suggests that this functionality is working even though there is a caveating final sentence clarifying that is not the case. I have suggested alternative wording that makes it clear that although the functionality has been added to the service it is not yet functional in the current infrastructure.
```suggestion
### Auto-Deployment of secrets
Functionality has been added to the nomad plan so that when the secrets are deployed to Vault, this will automatically cause Nomad to trigger a redeployment of the application to pick up the new secrets. Please note that this functionality does not appear to work with the current nomad/vault versions, but if these are upgraded it may then become functional. 

```
